### PR TITLE
UIOR-9 BigTest skeleteon and config

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**

### CODE REVIEW

**Comment by craigmcnally** [comment]
Hey guys, I added Matt Jones and Sobha Duvvuri as reviewers.  They have more experience with UI tests and Big Test than I do.

**Comment by ttamj** [comment]
@aliaksei-chumakou, The latest Stripes CLI v1.6 now automates the creation of the BigTest boilerplate.  This was just merged yesterday and released today.  I recommend updating your devDependency on stripes-cli to `^1.6.0` and running the newly added `stripes app bigtest` command to compare the boilerplate output for any differences.  At minimum should be able omit `.stripesclirc.js` from your repo as the v1.6 CLI now takes care of the `--mirage` option for you.  See the related Stripes CLI PR (https://github.com/folio-org/stripes-cli/pull/129) for reference.

**Comment by sduvvuri-ebsco** [comment]
@aliaksei-chumakou: While the BigTest config itself looks good, currently, it looks like all 4 tests are failing because of errors from mirage - mirage server is not configured to mock these particular requests.

**Comment by aliaksei-chumakou** [comment]
Jenkins please retry a build

**Comment by aliaksei-chumakou** [comment]
Jenkins please retry a build

File: package.json [hunk 1]
```diff
@@ -200,18 +200,28 @@
...
   },
   "devDependencies": {
+    "@bigtest/interactor": "^0.9.0",
+    "@bigtest/mirage": "^0.0.1",
+    "@bigtest/mocha": "^0.5.1",
+    "@bigtest/react": "^0.1.2",
     "@folio/eslint-config-stripes": "^3.2.1",
     "@folio/stripes": "^1.0.0",
+    "@folio/stripes-cli": "^1.6.0",
+    "@folio/stripes-core": "^2.15.4000415",
```
Comment (ttamj) [inline package.json]: Please use version `^12.15.4` of stripes-core.

This very large patch segment (`4000415`) is a sign that your dependency was added against the FOLIO CI registry, `npm-folioci`.  Unfortunately these inflated patch version numbers are specific to `npm-folioci` and not semver compatible with the `npm-folio` registry.  All `package.json` folio-scoped dependencies should reference versions available in `npm-folio`.

**Comment by aliaksei-chumakou** [comment]
Jenkins please retry a build

**Pull request merged**